db/schema.rb

```diff
@@ -264,23 +266,23 @@
     t.boolean "open_banking_consent"
     t.datetime "open_banking_consent_choice_at"
     t.string "own_home"
-    t.decimal "percentage_home"
     t.decimal "property_value", precision: 10, scale: 2
     t.string "shared_ownership"
     t.decimal "outstanding_mortgage_amount"
+    t.decimal "percentage_home"
```

Comment:
These fields moving suggest you didn't `rake db:schema:load` on starting this branch. The schema needs to be fixed, so that only the entries made in the migration modify the schema